api/lib/application/certification-courses/index.js

```diff
@@ -5,7 +5,31 @@ import { certificationCourseController } from './certification-course-controller
 import { identifiersType } from '../../../src/shared/domain/types/identifiers-type.js';
 
 const register = async function (server) {
-  server.route([
+  const adminRoutes = [
+    {
+      method: 'PATCH',
+      path: '/api/admin/certification-courses/{id}',
```

Comment:
Je propose d'en profiter pour appliquer une proposition de sécurité de Marc-Aurèle qui avait reçu un pourcentage favorable d'adoption et de mieux nommer nos `{id}`, ici donc

```suggestion
      path: '/api/admin/certification-courses/{certificationCourseId}',
```

Reply:
Par ailleurs il manque pour moi une indication sur l'ancienne route  `/api/certification-courses/{id}` que celle-ci est dépréciée (via un tag ou une mise à jour du descriptif)